dev/ci-test.sh

```diff
@@ -57,5 +58,21 @@ mvn --no-transfer-progress -Dtest=none -DwildcardSuites=org.apache.spark.ml.clus
 mvn --no-transfer-progress -Dtest=none -DwildcardSuites=org.apache.spark.ml.feature.IntelPCASuite test
```

Comment:
You can put it into SupportedSparkVersions as spark-3.0.0 instead of leave it outside the for loop.